Makefile

```diff
@@ -100,23 +100,25 @@ ifdef TAG
 ENV_VARS+=-e TAG="$(TAG)"
 endif
 
-RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):$(GID) --rm \
+RUN = $(CONTAINER_CLI) run --sig-proxy=true -u $(UID):$(GID) --rm \
```

Comment:
This is no need to start the container in interactive and terminal mode because this is a automation process. And it will cause some trouble if the caller process didn't get assigned to a tty, so I just removed them.

Reply:
I didn't find the "make shell" command and if I execute "make shell" in Istio src root. I got "make: *** No rule to make target 'shell'.  Stop." error. 

The "-t -i" options are used for user inputting. This is a automation process, no inputs needed from user. Removing this option doesn't affect shell script executing within docker container.

Reply:
Its in another repo, there is a pr out to add the Target to all repos. I don't think there is any reason we need to limit this to not user input

Reply:
Okay, this patch can be ignored.